css/search.css

```diff
@@ -0,0 +1,96 @@
+   * {
+  box-sizing: border-box;
+}
+.search {
+  width: 100px;
+  height: 100px;
+  margin: 40px auto 0;
+  background-color: #00FEDE;
+  position: relative;
+  overflow: hidden;
+  transition: all 0.5s ease;
+  border-radius: 70px;
+}
+.search:before {
+  content: '';
+  display: block;
+  width: 3px;
+  height: 100%;
+  position: relative;
+  background-color: #00FEDE;
+  transition: all 0.5s ease;
+}
+.search.open {
+  width: 420px;
+  border-radius: 70px;
```

Comment:
You have already declared the border-radius on the .search div so it already do have it from there,  you don't have to call the property in here as well